$MirOS: ports/misc/screen/patches/patch-config_h_in,v 1.1 2008/03/12 23:00:44 tg Exp $
$OpenBSD: shmpatch-Makefile_in,v 1.1 2003/11/22 00:56:58 espie Exp $
--- config.h.in.orig	Mon Oct 23 13:06:09 2006
+++ config.h.in	Sat Jan  9 17:49:27 2010
@@ -661,3 +661,12 @@
 #undef PTYRANGE0
 #undef PTYRANGE1
 
+#undef HAVE_UTIL_H
+#undef HAVE_LIBUTIL_H
+
+/*
+ * define IPC_EXPORT_IMAGE to allow an other program to get the screen
+ * content through shared memory and IPC. This is used e.g. for braille
+ * and speech software.
+ */
+#undef IPC_EXPORT_IMAGE
